waiter/src/waiter/client_tools.clj

```diff
@@ -143,9 +142,8 @@
 (defn make-http-client
   "Instantiates and returns a new HttpClient with a cookie store"
   []
-  (let [client (http/client)
-        cookie-store (HttpCookieStore.)
-        _ (.setCookieStore ^HttpClient client cookie-store)]
+  (let [client (http/client)]
+    (.setCookieStore client (HttpCookieStore$Empty.))
```

Comment:
I think we should modify the docstring to reflect the change

Reply:
Done